src/app/pcasts/controllers/facebook_sign_in_controller.py

```diff
@@ -0,0 +1,24 @@
+from app.pcasts.utils import facebook_utils
+from . import *
+
+class FacebookSignInController(AppDevController):
+
+  def get_path(self):
+    return '/users/facebook_sign_in/'
+
+  def get_methods(self):
+    return ['POST']
+
+  def content(self, **kwargs):
+    token = request.args['access_token']
```

Comment:
Let's make this part of the HTTP request body, rather than URL parameter.  Sending this as a URL parameter in plain text is dangerous.  We should also probably change this for our google sign in endpoint, as it was originally written quickly during the genesis of this project.

Reply:
Will handle this separately #132